.changeset/rare-zoos-flow.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
You can remove this file. I don't think this needs to be in the changelog as it is neither a fix nor a feature.